tasks/prepare-fbc-release/prepare-fbc-release.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: prepare-fbc-release
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task for prepration of FBC release this task will
```

Comment:
Why not copy&paste the description from the README? I think that's the de facto standard in this repo.

Reply:
Fixed, thanks!